bin/regression_stats.py

```diff
@@ -126,7 +126,7 @@ def str_and_colorize(left_side, right_side):
 
         row = [key]
         pairs = [
-            str_and_colorize(max(timing1.times), max(timing2.times)),
+            str_and_colorize(timing1.best_time(), timing2.best_time()),
```

Comment:
This should be using the Timing method rather than performing the operation itself. Further, we should use the best time rather than the worst time.